plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -92,20 +92,29 @@
     icon: Asset
     label: IntlString
   }
-  const statuses: Array<IPair> =
```

Comment:
I would prefer to have one Record with <EnumValue, { icon: Asset, label: IntlString}> instead of array.